src/components/app/Home.js

```diff
@@ -251,7 +251,7 @@ class Home extends PureComponent {
             <InstallButton name="Gecko Profiler" xpiURL={LEGACY_ADDON_URL}>
               Gecko Profiler Add-on
             </InstallButton>. Then use the button added to the browser, or use
-            the following shortcuts to record a profile. The button's icon is
+            the following shortcuts to record a profile. The button’s icon is
```

Comment:
from https://github.com/yannickcr/eslint-plugin-react/issues/894#issuecomment-252446640, on Mac you'd use `⌥-]` or `⌥-}`. On my Linux, it's right-alt + g, but I think it's layout-dependent :)